msvc-full-features/Cataclysm-common.props

```diff
@@ -67,6 +88,56 @@
       <GenerateDebugInformation>DebugFull</GenerateDebugInformation>
       <StripPrivateSymbols>$(OutDir)$(TargetName).stripped.pdb</StripPrivateSymbols>
     </Link>
+    <Link Condition="$(_CDDA_USE_LLD_LINK)">
+      <AdditionalLibraryDirectories Condition="$(Configuration.StartsWith(Debug))">
+        $(ProjectDir)vcpkg_installed\$(VcpkgTripet)\$(VcpkgTriplet)\debug\lib;%(AdditionalLibraryDirectories)
+      </AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories Condition="$(Configuration.StartsWith(Release))">
+        $(ProjectDir)vcpkg_installed\$(VcpkgTripet)\$(VcpkgTriplet)\lib;%(AdditionalLibraryDirectories)
+      </AdditionalLibraryDirectories>
+      <AdditionalDependencies>
+        brotlicommon-static.lib;
```

Comment:
I had to remove `-static` to link on latest vcpkg master.

Reply:
Unfortunately I haven't figured out a way to get this to figure out whether the vcpkg checkout is 'new enough' or not so that it can name the libs differently. So for now I keep it up to date with the documented 'known good' vcpkg revision.